op-challenger/metrics/metrics.go

```diff
@@ -93,6 +97,13 @@ func NewMetrics() *Metrics {
 				[]float64{1.0, 10.0},
 				prometheus.ExponentialBuckets(30.0, 2.0, 14)...),
 		}),
+		gameClaimCount: *factory.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
If it weren't for the fact that L1 can reorg & claims could disappear, this would be better as a counter (always increasing) rather than a gauge, but because the number can go down, we need to use it.